archeryutils/targets.py

```diff
@@ -18,20 +22,69 @@
     "Beiter_hit_miss",
     "Worcester",
     "Worcester_2_ring",
+    "Custom",
 ]
 
+# TypeAlias (annotate explicitly in py3.10+)
+#: A mapping of a target ring diameter to the score for hitting that ring.
+#: Typically ordered from smallest to largest rings but this is not necessary.
+FaceSpec = Mapping[float, int]
+
+_rnd6 = partial(round, ndigits=6)
+
+
+class Quantity(NamedTuple):
+    """
+    Dataclass for a quantity with units.
+
+    Can be used in place of a plain tuple of (value, units)
+
+    Parameters
+    ----------
+    value: float
+        Scalar value of quantity
+    units: str
+        Units of quantity
+
+    Notes
+    -----
+    It is recommened to use the `Quantity` type when passing specifyinging lengths
+    in _archeryutils_ for explictness and readability, and to ensure the expected units
+    are indeed being used downstream. Default units are assumed for convinience in
+    interactive use but this could cause breakages if the default unit changes
+    in the future.
+
+    Examples
+    --------
+    Define as a simple tuple:
+
+    >>> worcester_distance = au.Quantity(80, "yard")
+
+    Or with named keyword arguments:
+
+    >>> worcester_target_size = au.Quantity(value=122, units="cm")
+
+    These can then be passed on to any function that accepts a Quantity like tuple:
+
+    >>> worcester_target = au.Target(
+    ...     "Worcester",
+    ...     diameter=worcester_target_size,
+    ...     distance=worcester_distance,
+    ...     indoor=True,
+    ... )
+    """
+
+    value: float
+    units: str
+
 
 class Target:
     """
     Class to represent a target.
 
     Parameters
     ----------
-    scoring_system : {\
-        ``"5_zone"`` ``"10_zone"`` ``"10_zone_compound"`` ``"10_zone_6_ring"``\
-        ``"10_zone_5_ring"`` ``"10_zone_5_ring_compound"`` ``"WA_field"``\
-        ``"IFAA_field"`` ``"IFAA_field_expert"`` ``"Beiter_hit_miss"`` ``"Worcester"``\
-        ``"Worcester_2_ring"``}
+    scoring_system : ScoringSystem
```

Comment:
In many ways this is much cleaner, but it is now buried away that this is expected to be a string value matching one from the Literal, so perhaps make that explicit in the variable description?